src/gui/tray/UserModel.cpp

```diff
@@ -328,6 +351,10 @@ void User::slotProgressInfo(const QString &folder, const ProgressInfo &progress)
         const auto &engine = f->syncEngine();
         const auto style = engine.lastLocalDiscoveryStyle();
         foreach (Activity activity, _activityModel->errorsList()) {
+            if (activity._expireAtMsecs > 0) {
```

Comment:
that should be `!= -1` if I read the default value for `_expireAtMsecs`

Reply:
> that should be `!= -1` if I read the default value for `_expireAtMsecs`

@mgallien yes, but, I always check for non-negative and non-zero, isn't it more reliable? if someone changes default to `0` or `-2`, this code will not work.

Reply:
@mgallien anyways, I've decided to change it.